docs/docs/components/guidelines/README.md

```diff
@@ -426,7 +436,9 @@ In the interest of consistency, use the following naming patterns when defining
 | `$.interface.timer` | `timer`                            |
 | `$.service.db`      | `db`                               |
 
-### Sources
```

Comment:
I changed the name of this header because we used the "Sources" header label in two places, so this disambiguates it from the other.